third_party/java/nanohttpd/BUILD

```diff
@@ -0,0 +1,13 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Do we need headers on these?

Reply:
None of the other //third_party BUILD files have them, so I'm assuming not.